src/css/angular-tooltips.css

```diff
@@ -36,44 +36,87 @@
   transition: opacity 0.2s linear;
   opacity: 1;
 }
+._720kb-tooltip-caret:before {
```

Comment:
They are the same. The color of triangle is darker because of opacity.
How about set “after” no color? it makes the triangle look like what it is before.